l2geth/consensus/clique/clique.go

```diff
@@ -640,7 +640,7 @@ func (c *Clique) Seal(chain consensus.ChainReader, block *types.Block, results c
 
 		log.Trace("Out-of-turn signing requested", "wiggle", common.PrettyDuration(wiggle))
 	}
-	if vm.UsingOVM {
+	if rcfg.UsingOVM {
```

Comment:
Since you have a diff here, this could use a comment describing why this diff exists. Something along the lines of:
```
Set the delay to 0 when using the OVM so that blocks are always
produced instantly. When running in a network, the delay prevents
the creation of invalid blocks
```